app/src/main/java/org/mozilla/reference/browser/browser/ToolbarIntegration.kt

```diff
@@ -109,6 +110,10 @@ class ToolbarIntegration(
             addHistoryStorageProvider(historyStorage)
             addDomainProvider(shippedDomainsProvider)
         }
+
+        toolbar.urlBoxView = View(context).apply {
+            background = context.resources.getDrawable(R.drawable.url_background, context.theme)
```

Comment:
I like this theme! We can remove the background in CT in later PR since I'm guessing this isn't intentional.

Fenix has this issue as well: https://github.com/mozilla-mobile/fenix/issues/535

<img width="896" alt="screen shot 2019-02-19 at 11 15 33 am" src="https://user-images.githubusercontent.com/1370580/53029859-bbcb0180-3437-11e9-97fc-d7e9070abee6.png">

Reply:
Oh, yes.

Reply:
Fix one CT thing, create a new one! :)

Reply:
I'll open a new one for this. This is going to annoy me next. :)